de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/progress/ConsoleProgressIndicator.java

```diff
@@ -0,0 +1,100 @@
+package de.fu_berlin.inf.dpp.server.progress;
+
+import de.fu_berlin.inf.dpp.monitoring.IProgressMonitor;
+
+import java.io.PrintStream;
+
+public class ConsoleProgressIndicator implements IProgressMonitor {
```

Comment:
JavaDoc for the purpose of this class. To me it looks like this is more debug specific. Especially the worked() method might flood the output stream.